js/clinicalTimeline.js

```diff
@@ -156,7 +155,7 @@ var clinicalTimeline = (function(){
     d3.select(divId + " svg")
       .insert("rect", ".timeline-label")
       .attr("width", 130)
-      .attr("height", gBoundingBox.height - 15)
+      .attr("height", 245)
```

Comment:
@inodb , where is it used elsewhere? Both lines you mentioned are 158 in clinicalTimeline.js